<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="CFoldersView" name="folders" native="true"/>
       <widget class="CFilterFromEmailView" name="email" native="true"/>
      </widget>
      <widget class="CRulesView" name="rules" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSelectServer"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRules">
    <property name="title">
     <string>&amp;Rules</string>
    </property>
    <widget class="QMenu" name="menuRun">
     <property name="title">
      <string>Run</string>
     </property>
     <addaction name="actionRunSelectedRule"/>
     <addaction name="actionRunSelectedRuleOnSelectedFolder"/>
     <addaction name="separator"/>
     <addaction name="actionRunAllRules"/>
     <addaction name="actionRunAllRulesOnSelectedFolder"/>
     <addaction name="separator"/>
     <addaction name="actionRunAllRulesOnJunkFolder"/>
     <addaction name="actionRunAllRulesOnTrashFolder"/>
    </widget>
    <addaction name="actionAddRule"/>
    <addaction name="actionAddToSelectedRule"/>
    <addaction name="separator"/>
    <addaction name="menuRun"/>
    <addaction name="separator"/>
    <addaction name="actionRuleEnabled"/>
    <addaction name="actionDeleteRule"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionOnlyProcessUnread"/>
    <addaction name="actionProcessAllEmailWhenLessThan200Emails"/>
    <addaction name="actionOnlyProcessTheFirst500Emails"/>
    <addaction name="actionIncludeJunkFolderWhenRunningOnAllFolders"/>
    <addaction name="actionIncludeDeletedFolderWhenRunningOnAllFolders"/>
    <addaction name="separator"/>
    <addaction name="actionDisableRatherThanDeleteRules"/>
   </widget>
   <widget class="QMenu" name="menuFolders">
    <property name="title">
     <string>F&amp;olders</string>
    </property>
    <addaction name="actionAddFolderForSelectedEmail"/>
   </widget>
   <widget class="QMenu" name="menuReload">
    <property name="title">
     <string>R&amp;eload</string>
    </property>
    <addaction name="actionReloadAllData"/>
    <addaction name="separator"/>
    <addaction name="actionReloadEmail"/>
    <addaction name="actionReloadFolders"/>
    <addaction name="actionReloadRules"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRenameRules"/>
    <addaction name="actionSortRules"/>
    <addaction name="separator"/>
    <addaction name="actionMergeRules"/>
    <addaction name="separator"/>
    <addaction name="actionEnableAllRules"/>
    <addaction name="actionDeleteAllDisabledRules"/>
    <addaction name="separator"/>
    <addaction name="actionFindEmptyFolders"/>
    <addaction name="separator"/>
    <addaction name="actionEmptyJunkFolder"/>
    <addaction name="actionEmptyTrash"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuReload"/>
   <addaction name="menuFolders"/>
   <addaction name="menuRules"/>
   <addaction name="menuTools_2"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectServer"/>
   <addaction name="separator"/>
   <addaction name="actionReloadAllData"/>
   <addaction name="separator"/>
   <addaction name="actionReloadEmail"/>
   <addaction name="actionReloadFolders"/>
   <addaction name="actionReloadRules"/>
   <addaction name="separator"/>
   <addaction name="actionAddFolderForSelectedEmail"/>
   <addaction name="separator"/>
   <addaction name="actionAddRule"/>
   <addaction name="actionAddToSelectedRule"/>
   <addaction name="separator"/>
   <addaction name="actionRunAllRulesOnSelectedFolder"/>
  </widget>
  <action name="actionSelectServer">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/server.png</normaloff>:/resources/server.png</iconset>
   </property>
   <property name="text">
    <string>Select Server...</string>
   </property>
  </action>
  <action name="actionReloadAllData">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload All</string>
   </property>
   <property name="iconText">
    <string>All</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionAddRule">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/new.png</normaloff>:/resources/new.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;Rule...</string>
   </property>
  </action>
  <action name="actionOnlyProcessUnread">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Only Process Unread?</string>
   </property>
   <property name="toolTip">
    <string>Only Process Unread?</string>
   </property>
  </action>
  <action name="actionReloadEmail">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload Email</string>
   </property>
   <property name="iconText">
    <string>Email</string>
   </property>
  </action>
  <action name="actionRunSelectedRule">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/run.png</normaloff>:/resources/run.png</iconset>
   </property>
   <property name="text">
    <string>Run Selected Rule</string>
   </property>
  </action>
  <action name="actionAddToSelectedRule">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/modify.png</normaloff>:/resources/modify.png</iconset>
   </property>
   <property name="text">
    <string>Add to Selected Rule</string>
   </property>
  </action>
  <action name="actionReloadRules">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload Rules</string>
   </property>
   <property name="iconText">
    <string>Rules</string>
   </property>
  </action>
  <action name="actionReloadFolders">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/reload.png</normaloff>:/resources/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload Folders</string>
   </property>
   <property name="iconText">
    <string>Folders</string>
   </property>
  </action>
  <action name="actionSortRules">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/sort.png</normaloff>:/resources/sort.png</iconset>
   </property>
   <property name="text">
    <string>Sort Rules...</string>
   </property>
  </action>
  <action name="actionRenameRules">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/rename.png</normaloff>:/resources/rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename Rules...</string>
   </property>
  </action>
  <action name="actionMergeRules">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/merge.png</normaloff>:/resources/merge.png</iconset>
   </property>
   <property name="text">
    <string>Merge Rules...</string>
   </property>
  </action>
  <action name="actionProcessAllEmailWhenLessThan200Emails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Process all Email when less than 200 Emails?</string>
   </property>
   <property name="toolTip">
    <string>Process All Email when folder has less than 200 Emails?</string>
   </property>
  </action>
  <action name="actionRunAllRules">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/run.png</normaloff>:/resources/run.png</iconset>
   </property>
   <property name="text">
    <string>Run All Rules</string>
   </property>
  </action>
  <action name="actionAddFolderForSelectedEmail">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/folder.png</normaloff>:/resources/folder.png</iconset>
   </property>
   <property name="text">
    <string>Add Folder for Selected Email</string>
   </property>
  </action>
  <action name="actionEnableAllRules">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/enable.png</normaloff>:/resources/enable.png</iconset>
   </property>
   <property name="text">
    <string>Enable All Rules...</string>
   </property>
  </action>
  <action name="actionRunAllRulesOnSelectedFolder">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/run.png</normaloff>:/resources/run.png</iconset>
   </property>
   <property name="text">
    <string>Run All Rules on Selected Folder</string>
   </property>
  </action>
  <action name="actionRunSelectedRuleOnSelectedFolder">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/resources/run.png</normaloff>:/resources/run.png</iconset>
   </property>
   <property name="text">
    <string>Run Selected Rule on Selected Folder</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionEmptyJunkFolder">
   <property name="text">
    <string>Empty Junk Folder</string>
   </property>
  </action>
  <action name="actionEmptyTrash">
   <property name="text">
    <string>Empty Trash</string>
   </property>
  </action>
  <action name="actionIncludeJunkFolderWhenRunningOnAllFolders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Include Junk Folder when Running on All Folders?</string>
   </property>
  </action>
  <action name="actionDisableRatherThanDeleteRules">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disable rather than Delete Rules?</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionOnlyProcessTheFirst500Emails">
   <property name="text">
    <string>Only Process the first 500 Emails?</string>
   </property>
  </action>
  <action name="actionIncludeDeletedFolderWhenRunningOnAllFolders">
   <property name="text">
    <string>Include Deleted Folder when Running on All Folders?</string>
   </property>
  </action>
  <action name="actionRuleEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rule Enabled?</string>
   </property>
  </action>
  <action name="actionDeleteRule">
   <property name="text">
    <string>Delete Rule</string>
   </property>
  </action>
  <action name="actionRunAllRulesOnJunkFolder">
   <property name="text">
    <string>Run All Rules on Junk Folder</string>
   </property>
  </action>
  <action name="actionRunAllRulesOnTrashFolder">
   <property name="text">
    <string>Run All Rules on Trash Folder</string>
   </property>
  </action>
  <action name="actionDeleteAllDisabledRules">
   <property name="text">
    <string>Delete All Disabled Rules...</string>
   </property>
  </action>
  <action name="actionFindEmptyFolders">
   <property name="text">
    <string>Find Empty Folders...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CFoldersView</class>
   <extends>QWidget</extends>
   <header>FoldersView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CFilterFromEmailView</class>
   <extends>QWidget</extends>
   <header>FilterFromEmailView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CRulesView</class>
   <extends>QWidget</extends>
   <header>RulesView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
